dev/tools/gen_keycodes/data/keyboard_key.tmpl

```diff
@@ -46,75 +46,11 @@ abstract class KeyboardKey with Diagnosticable {
 /// look at the physical key to make sure that regardless of the character the
 /// key produces, you got the key that is in that location on the keyboard.
 ///
-/// {@tool dartpad --template=stateful_widget_scaffold}
+/// {@tool dartpad}
 /// This example shows how to detect if the user has selected the logical "Q"
 /// key.
 ///
-/// ```dart imports
-/// import 'package:flutter/foundation.dart';
-/// import 'package:flutter/services.dart';
-/// ```
-///
-/// ```dart
-/// // The node used to request the keyboard focus.
-/// final FocusNode _focusNode = FocusNode();
-/// // The message to display.
-/// String? _message;
-///
-/// // Focus nodes need to be disposed.
-/// @override
-/// void dispose() {
-///   _focusNode.dispose();
-///   super.dispose();
-/// }
-///
-/// // Handles the key events from the RawKeyboardListener and update the
-/// // _message.
-/// void _handleKeyEvent(RawKeyEvent event) {
-///   setState(() {
-///     if (event.logicalKey == LogicalKeyboardKey.keyQ) {
-///       _message = 'Pressed the "Q" key!';
-///     } else {
-///       if (kReleaseMode) {
-///         _message = 'Not a Q: Pressed 0x${event.logicalKey.keyId.toRadixString(16)}';
-///       } else {
-///         // The debugName will only print useful information in debug mode.
-///         _message = 'Not a Q: Pressed ${event.logicalKey.debugName}';
-///       }
-///     }
-///   });
-/// }
-///
-/// @override
-/// Widget build(BuildContext context) {
-///   final TextTheme textTheme = Theme.of(context).textTheme;
-///   return Container(
-///     color: Colors.white,
-///     alignment: Alignment.center,
-///     child: DefaultTextStyle(
-///       style: textTheme.headline4!,
-///       child: RawKeyboardListener(
-///         focusNode: _focusNode,
-///         onKey: _handleKeyEvent,
-///         child: AnimatedBuilder(
-///           animation: _focusNode,
-///           builder: (BuildContext context, Widget? child) {
-///             if (!_focusNode.hasFocus) {
-///               return GestureDetector(
-///                 onTap: () {
-///                   FocusScope.of(context).requestFocus(_focusNode);
-///                 },
-///                 child: const Text('Tap to focus'),
-///               );
-///             }
-///             return Text(_message ?? 'Press a key');
-///           },
-///         ),
-///       ),
-///     ),
-///   );
-/// }
-/// ```
+/// ** See code in examples/api/lib/services/keyboard_key/logical_keyboard_key.0.dart **
```

Comment:
Thank you for converting this, I missed it during my work moving the examples out.